libs/ett/core/shell/src/lib/ett-primary-navigation.component.ts

```diff
@@ -31,20 +34,71 @@ const selector = 'ett-primary-navigation';
   selector,
   styles: [
     `
+      /**
+       * 1. Add active indicator to the active link.
+       * 2. Highlight the active link.
+       */
+
       ${selector} {
         display: block;
+
+        .active-indicator /* [1] */ {
+          --size: var(--watt-space-xs);
+          display: none;
+
+          width: var(--size);
+          height: 100%;
+          margin-right: calc(var(--watt-space-m) - var(--size));
+
+          background-color: var(--watt-color-focus);
+        }
+
+        a.is-active {
+          background-color: var(--watt-color-primary); // [2]
+          color: var(--watt-color-primary-contrast); // [2]
+
+          .active-indicator /* [1] */ {
+            display: block;
+          }
+
+          > .mat-list-item-content.mat-list-item-content /* [1] */ {
+            padding-left: 0;
+          }
+        }
       }
     `,
   ],
   template: `
     <mat-nav-list>
-      <a mat-list-item routerLink="/dashboard" routerLinkActive="active">
+      <a mat-list-item routerLink="/dashboard" routerLinkActive="is-active">
```

Comment:
question:

How come we have a div to show the active item? Can a class be added to the a tag from material?

Reply:
I couldn't make it work with a pseudo element. Any suggestions?

Reply:
Hmm, I was thinking if we could just add the class "active" on the a tag from A.M - And style it in the component, would that work, or would A.M. override that?

Reply:
Removed indicator element in favor of a CSS only solution.